### DISCUSSION TITLE
[resolved] Czy jakakolwiek metoda API zwraca informację o usłudze SMART?

### DISCUSSION THREAD

**Discussion by javaguru-pl** [allegro/allegro-api#4536] [Suggestions - resolved]
Witam, pytanie podobne do #4471 - ale tym razem chodzi o **SMART**. Czy jakakolwiek metoda API zwraca informacje o aktywnej (lub nieaktywnej) usłudze SMART dla naszego własnego konta lub dla naszych własnych aukcji?

Ponownie przejrzałem metody API, ale nic takiego nie znalazłem.

Z góry dziękuję za szybką odpowiedź.

**Comment by Lukasz-Zurek** [comment]
Niestety nie mamy tego typu zasobów.

**Comment by javaguru-pl** [comment]
A czy jest możliwość dodania takich informacji (zarówno SMART jaki i SuperSprzedawca).
Myślę że najlepiej do metody /me.

**Comment by Lukasz-Zurek** [comment]
Dziękuję za sugestię. Zgodnie z [naszym newsem](https://github.com/allegro/allegro-api/issues/4337) przenoszę wątek do zakładki [Discussions](https://github.com/allegro/allegro-api/discussions).

**Comment by MartaNowaczyk** [comment]
Rozwiązania dotyczące [klasyfikacji SMART](https://developer.allegro.pl/news/udostepnilismy-nowe-zasoby-dzieki-ktorym-sprawdzisz-klasyfikacje-ofert-i-sprzedawcy-w-programie-allegro-smart-D7KrOL5YBIw) są dostępne:

- [GET /sale/offers/{offerId}/smart](https://developer.allegro.pl/documentation/#operation/getOfferSmartClassificationGET) - sprawdź klasyfikację oferty w programie Allegro Smart!,
- [GET /sale/smart](https://developer.allegro.pl/documentation/#operation/getSellerSmartClassificationGET) - sprawdź klasyfikację sprzedawcy w programie Allegro Smart!.

Zmieniam status na resolved.

**Reply by javaguru-pl** [reply]
Witam Państwa, bardzo dziękuję za dodanie w/w metod dot. SMART. Mam jednak pytanie - czy zmiana statusu SMART dla konkretnej oferty - powoduje emisję zdarzenia OFFER_CHANGED? (lub jakiegokolwiek innego zdarzenia w dzienniku zdarzeń? - jeśli tak to jakiego?)
Z góry dziękuję za odpowiedź.

**Reply by MartaNowaczyk** [reply]
Witam @javaguru-pl, nie wysyłamy zdarzenia po zmianie statusu SMART dla oferty.
